src/containers/item-details/item-details.tsx

```diff
@@ -21,6 +21,8 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
 
   const isOwner: boolean = address === item.owner.address;
 
+  const isAuthor: boolean = address === item.author.address;
```

Comment:
It is cool to explicit type the data types, but on this case this is obvious and not necessary. It is like saying `const a:number = 2;`